streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -1464,6 +1464,10 @@ protected byte uniqueField() {
         return uniqueField;
     }
 
+    protected Map<String, String> clientTags() {
+        return clientTags;
```

Comment:
This seems to be only used in its own test.

Reply:
Sorry, my bad. I misinterpreted the code.

Reply:
It is indeed only used in tests. But seems like that's a common pattern in the `StreamsPartitionAssignor`. Followed the same approach.